site-src/api-types/backendtlspolicy.md

```diff
@@ -0,0 +1,144 @@
+```mermaid
+flowchart LR
+    backendTLSPolicy[["<b>backendTLSPolicy</b> <hr><align=left>BackendTLSPolicySpec: spec<br>PolicyStatus: status</align>"]]
+    spec[["<b>spec</b><hr>PolicyTargetReferenceWithSectionName: targetRef <br> BackendTLSPolicyConfig: tls"]]
+    status[["<b>status</b><hr>[ ]PolicyAncestorStatus: ancestors"]]
+    tls[["<b>tls</b><hr>LocalObjectReference: caCertRefs<br>wellKnownCACertType: wellKnownCACerts<br>PreciseHostname: hostname"]]
+    ancestorStatus[["<b>ancestors</b><hr>AncestorRef: parentReference<br>GatewayController: controllerName<br>[]Condition: conditions"]]
+    targetRef[[<b>targetRef</b><hr>]]
+    service["<b>service</>"]
+    backendTLSPolicy -->spec
+    backendTLSPolicy -->status
+    spec -->targetRef & tls
+    status -->ancestorStatus
+    targetRef -->service
+    note[<em>choose only one<hr> caCertRef OR wellKnownCACerts</em>]
+    style note fill:#fff
+    tls -.- note
+```
+
+The following illustrates a BackendTLSPolicy that configures TLS for a Service serving a backend:
+```mermaid
+flowchart LR
+    client(["client"])
+    gateway["Gateway"]
+    style gateway fill:#02f,color:#fff
+    httproute["HTTP<BR>Route"]
+    style httproute fill:#02f,color:#fff
+    service["Service"]
+    style service fill:#02f,color:#fff
+    pod1["Pod"]
+    style pod1 fill:#02f,color:#fff
+    pod2["Pod"]
+    style pod2 fill:#02f,color:#fff
+    client -.->|HTTP <br> request| gateway
+    gateway --> httproute
+    httproute -.->|BackendTLSPolicy|service
+    service --> pod1 & pod2
+```
+
+### Targeting backends
+
+A BackendTLSPolicy targets a backend Pod (or set of Pods) via a TargetRef to a Service.  This TargetRef is a
+required object reference that specifies a Service by its Name, Kind (Service), and optionally its Namespace and Group.
+TargetRef identifies the Service for which your HTTPRoute requires TLS.
+
+!!! info "Restrictions"
+
+    - Cross-namespace certificate references are not allowed.
+
+### BackendTLSPolicyConfig
+
+A BackendTLSPolicyConfig is the specification for the BackendTLSPolicy and defines the configuration for TLS,
+including hostname (for server name indication) and certificates.
+
+#### Hostname
+
+Hostname defines the server name indication (SNI) the Gateway should use in order to connect to the backend, and must
+match the certificate served by the backend pod. A hostname is the fully qualified domain name of a network host, as
+defined by [RFC 3986][rfc-3986]. Note the following deviations from the “host” part of the URI as defined in the RFC:
+
+- IP addresses are not allowed.
+
+Also note:
+
+!!! info "Restrictions"
+
+    - Wildcard hostnames are not allowed.
+
+#### Certificates
+
+The BackendTLSPolicyConfig must contain a certificate reference of some kind, and contains two ways to configure the
+certificate to use for backend TLS, CACertRefs and WellKnownCACerts.  Only one of these may be used per
+BackendTLSPolicyConfig.
+
+##### CaCertRefs
```

Comment:
Potential follow up - this is currently styled in all caps (https://deploy-preview-2547--kubernetes-sigs-gateway-api.netlify.app/api-types/backendtlspolicy/). It might be better to just go with bold text here instead of a heading to avoid that styling.